instrumentation/message-handler/aws/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+dependencies {
+  compileOnly("io.opentelemetry:opentelemetry-sdk")
+
+  implementation(project(":instrumentation:message-handler:core"))
+
+  library("software.amazon.awssdk:sqs:2.2.0")
```

Comment:
Following up on https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8331#discussion_r1214018170:

I think an AWS Lambda implementation of the message handler that is then also used in the implementation of the existing AWS Lamba instrumentaion could still be great to validate the design

Reply:
Due to a limitation, we won't be able to support that in our backend.
We can talk more in private in Slack if you want, but I don't want to discuss it in a public channel.
@Oberon00